index.html

```diff
@@ -0,0 +1,146 @@
+
+          <li class="nav-item">
+            <a class="nav-link" href="#features">Features</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#video">How it Works</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#pricing">Pricing</a>
+          </li>
+
+        </ul>
+      </div>
+    </nav>
+
+
+  </header>
+
+  <h3>Handcrafted, home-made masterpieces</h3> <br>
+  <form id="form" action="https://www.freecodecamp.com/email-submit">
+    <input id="email" type="email" placeholder="enter your email address"> <br> <br>
+    <input id="submit" type="submit" value="Get Started">
+  </form> <br>
+
+  <div id="features" class="container">
+    <div class="grid">
+      <div class="icon"></div>
+      <div class="desc">
+        <h2>Premium Materials</h2>
+        <p>
+          Our trombones use the shiniest brass which is sourced locally. This
+          will increase the longevity of your purchase.
+        </p>
+      </div>
+    </div>
+    <div class="grid">
+      <div class="icon"></div>
+      <div class="desc">
+        <h2>Fast Shipping</h2>
+        <p>
+          We make sure you recieve your trombone as soon as we have finished making it. We also provide free
+          returns if you are not satisfied.
+        </p>
+      </div>
+    </div>
+    <div class="grid">
+      <div class="icon"></div>
+      <div class="desc">
+        <h2>Quality Assurance</h2>
+        <p>
+          For every purchase you make, we will ensure there are no damages or faults and we will check and
+          test the pitch of your instrument
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    <iframe id="video" width="420" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc">
+    </iframe>
+  </div> <br>
+
+  <div id="pricing-container">
+    <div id="pricing">
+      <header>TENOR TROMBONE</header>
+      <h4>$600</h4>
+      <ul>
+        <li>Lorem ipsum.</li>
+        <li>Lorem ipsum.</li>
+        <li>Lorem ipsum dollar.</li>
+        <li>Lorem ipsum.</li>
+      </ul>
+      <button type="submit">SELECT</button>
+    </div>
+    <div id="pricing2">
+      <header>BASS TROMBONE</header>
+      <h4>$900</h4>
+      <ul>
+        <li>Lorem ipsum.</li>
+        <li>Lorem ipsum.</li>
+        <li>Lorem ipsum dollar.</li>
+        <li>Lorem ipsum.</li>
+      </ul>
+      <button type="submit">SELECT</button>
+    </div>
+    <div id="pricing3">
+      <header>VALVE TROMBONE</header>
+      <h4>$1200</h4>
+      <ul>
+        <li>Lorem ipsum.</li>
+        <li>Lorem ipsum.</li>
+        <li>Lorem ipsum dollar.</li>
+        <li>Lorem ipsum.</li>
+      </ul>
+      <button type="submit">SELECT</button>
+    </div>
+  </div>
+
+  <br>
+
+
+  <div>
+    <a href="#header">Privacy</a>
+    <a href="#header">Terms</a>
+    <a href="mailto:hege@example.com">Contact</a>
+    <p>Copyright 2016, Original Trombones<br>
+  </div>
```

Comment:
It would make your code more semantic, if you replace `div` tags with `footer` tag. Since it is nice to set them in semantically.

Reply:
Kindly add the link to the semantic tags to give the students more understanding of what you are saying😉.